addons/web/static/src/js/framework/time.js

```diff
@@ -238,6 +238,19 @@ function strftime_to_moment_format (value) {
     return _normalize_format_cache[value];
 }
 
+/**
+ * Convert moment.js format to python strftime
+ *
+ * @param {String} value original format
+ */
+function moment_to_strftime_format(value) {
+    var inverse_normalize_format_table = _.invert(normalize_format_table);
```

Comment:
Ignoring that I see no reason for this function to exist, why isn't the inverse table computed statically, just once?